index.js

```diff
@@ -0,0 +1,311 @@
+    completed.push(toDo[index]);
+    createTaskField(toDo[index], completeField)
+    toDo.splice(index, 1);
+    toDoCount--
+    document.getElementById('toDo-count').innerHTML = ' (' + toDoCount + ')';
+    completedToDoCount++
+    document.getElementById('completed-count').innerHTML = ' (' + completedToDoCount + ')';
+    saveData();
+}
+
+function editTaskData() {
+    addTaskButton.style.display = "none";
+    editTaskButton.style.display = "inline";
+    let currentTask = this.closest('.list-group-item');
+    index = Array.from(currentTask.parentNode.children).indexOf(currentTask);
+    title.value = toDo[index].title;
+    text.value = toDo[index].text;
+    switch (toDo[index].priority) {
+        case 'Low priority':
+            document.getElementById('Low').checked = "true";
+            break;
+        case 'Medium priority':
+            document.getElementById('Medium').checked = "true";
+            break;
+        case 'High priority':
+            document.getElementById('High').checked = "true";
+            break;
+        default:
+            document.getElementById('Low').checked = "true";
+            break;
+    }
+
+}
+
+function saveEditChanges() {
+    let li = document.querySelectorAll('.list-group-item')[index];
+    let task = {
+        title: '',
+        text: '',
+        priority: '',
+        time: ''
+    }
+    task.title = title.value;
+    task.text = text.value;
+    task.priority = selectPriority() + ' priority';
+    task.time = setTaskTime();
+    toDo.splice(index, 1, task);
+    let newTitle = taskField.children[index].getElementsByTagName("h5")[0];
+    let newText = taskField.children[index].getElementsByTagName("p")[0];
+    let newPriority = taskField.children[index].getElementsByTagName("small")[0];
+    let newTime = taskField.children[index].getElementsByTagName("small")[1];
+    newTitle.innerHTML = task.title;
+    newText.innerHTML = task.text;
+    newPriority.innerHTML = selectPriority() + ' priority';
+    newTime.innerHTML = setTaskTime();
+    colorTask(li, selectPriority() + ' priority');
+    editTaskButton.setAttribute("data-dismiss", "modal");
+    saveData();
+}
+
+function deleteTaskData() {
+    let currentTask = this.closest('.list-group-item');
+    index = Array.from(currentTask.parentNode.children).indexOf(currentTask);
+    let type = this.closest('.list-group-item').parentNode.id;
+    if (type === "currentTasks") {
+        toDo.splice(index, 1);
+        taskField.removeChild(currentTask);
+        toDoCount--
+        document.getElementById('toDo-count').innerHTML = ' (' + toDoCount + ')';
+    } else {
+        completed.splice(index, 1);
+        completeField.removeChild(currentTask);
+        completedToDoCount--
+        document.getElementById('completed-count').innerHTML = ' (' + completedToDoCount + ')';
+    }
+    saveData();
+}
+
+function sortFromNew() {
+    toDo.sort(function(a, b) {
+        return new Date(a.date) - new Date(b.date);

```

Comment:
You don't need to create new Date in comparison. You can compare timestamps.

Reply:
Thank you!
fixed it!
```suggestion
        return a.date - b.date;
```